### PR TITLE
feat: add delete/domains command

### PR DESCRIPTION
**Pull request by PatrickMenoti**
**WHAT**
- Add delete domains command;
- Unit tests;

### CODE REVIEW

**Review by helio-neto — DISMISSED** [review]
🚀

File: pkg/cmd/delete/domains/domains.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+					{
+						Name:     "id",
+						Prompt:   &survey.Input{Message: "What is the id of the Domain you wish to delete?"},
+						Validate: survey.Required,
+					},
+				}
+
+				answer := ""
+
+				err := survey.Ask(qs, &answer)
```
[the remark this answers is not in the document]
Comment (PatrickMenoti) [reply pkg/cmd/delete/domains/domains.go:43]: Good catch!

File: messages/delete/domains/errors.go [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package domains
+
+import "errors"
+
+var (
+	ErrorConvertId          = errors.New("The application ID you provided is invalid. The value must be an integer. You may run the 'azion list domains' command to check your domain ID")
```
[the remark this answers is not in the document]
Comment (PatrickMenoti) [reply messages/delete/domains/errors.go:6]: oops

**Review by maxwelbm — APPROVED** [review]
Lgtm 🥇

**Review by helio-neto — APPROVED** [review]

**Pull request merged**